lib/sdf-server/src/service/public/change_sets.rs

```diff
@@ -39,7 +39,7 @@ pub fn routes(state: AppState) -> Router<AppState> {
             .route("/request_approval", post(request_approval))
             .route(
                 "/force_apply",
-                post(force_apply).layer(WorkspacePermissionLayer::new(
+                post(force_apply).route_layer(WorkspacePermissionLayer::new(
```

Comment:
To be clear, the problem before was the layer applied to all routes and you just want it on this route ("force_apply")

Reply:
It only applies to this route; I was getting a situation where this layer was running before the parent's `route_layer()` (which set up the workspace id for it). My hypothesis was that all layer()s run before all route_layer()s. Either way, this fixed the ordering issue.